functionalTest/test/specs/overview.js

```diff
@@ -7,6 +8,101 @@ describe("Covid-19 Dashboard", () => {
     expect(browser).toHaveTitle(title);
     expect(dashboard.imgLogo).toBeDisplayed();
     expect(dashboard.headingTitle).toHaveText("Scottish COVID-19 Statistics");
-    expect(dashboard.headlineBanner).toHaveTextContaining("Phase 3");
+    expect(dashboard.headlineBanner).toHaveTextContaining(
+      "Regional Restrictions to help fight the pandemic"
```

Comment:
This is the message for the regional insights page isn't it, not the overview page

Reply:
Changed to phase 3